app/views/badges/require.slim

```diff
@@ -3,13 +3,13 @@
     .col-md-12
       .form-container
         h3
-          | OBTER BADGE_
+          | OBTER #{t('badge').upcase.tr('á','Á')}_
```

Comment:
Why do you need to do that `tr` call? And since it's repeated maybe move it to a helper method and apply that transformation to the most common accented characters. Something like `tr('áàéó', 'ÁÀÉÓ')`